tests/test_mtu.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import pytest
+from netaddr import *
+import time
+import logging
+import requests
+from ptf_runner import ptf_runner
+from ansible_host import AnsibleHost
+from datetime import datetime
+
+
+@pytest.mark.parametrize("mtu", [1514,9114])
+def test_mtu(ansible_adhoc, testbed, duthost, ptfhost, mtu):
```

Comment:
The `ansible_adhoc` fixtures is not needed here. Suggest to remove them.

Reply:
Removed the ansible_adhoc